packages/opentelemetry-instrumentation-anthropic/opentelemetry/instrumentation/anthropic/config.py

```diff
@@ -0,0 +1,2 @@
+class Config:
+    enrich_token_usage = True
```

Comment:
A few comment on this Config:
1. I think it's best if its `False` by default. 
2. You also need to have an option to set it during `__init__` of the instrumentation (see the OpenAI instrumentation for an example).
3. You can then pass `True` to it when `traceloop_sdk` initialize it (see how we did it with the OpenAI instrumentation).